core/src/io/anuke/mindustry/net/Administration.java

```diff
@@ -126,7 +131,7 @@ public boolean unbanPlayerID(String id){
         info.banned = false;
         bannedIPs.removeAll(info.ips, false);
         save();
-
+        Events.fire(new PlayerUnbanEvent(Vars.playerGroup.find(p -> p.uuid.equals(id))));
```

Comment:
`player.uuid` can be null, causing a crash here. The condition should be inverted.

Reply:
What do you mean by inverting the condition?

Reply:
`id.equals(p.uuid)` instead of `p.uuid.equals(id)`